modules/carousel/jetpack-carousel.php

```diff
@@ -538,20 +538,20 @@ function post_attachment_comment() {
 			header('Content-type: text/javascript');
 
 		if ( empty( $_POST['nonce'] ) || ! wp_verify_nonce($_POST['nonce'], 'carousel_nonce') )
-			die( json_encode( array( 'error' => __( 'Nonce verification failed.', 'jetpack' ) ) ) );
+			die( json_encode( array( 'error' => esc_html__( 'Nonce verification failed.', 'jetpack' ) ) ) );
```

Comment:
I wouldn't bother escaping translations that will get output as an error with the `die` function.